lambdas/backend-api/src/__tests__/templates/infra/template-repository.test.ts

```diff
@@ -1534,65 +1531,49 @@ describe('templateRepository', () => {
       });
     });
 
-    describe('when proofing flag is disabled', () => {
-      test.each([
-        {
-          globalProofing: false,
-          clientProofing: false,
-        },
-        {
-          globalProofing: true,
-          clientProofing: false,
-        },
-        {
-          globalProofing: false,
-          clientProofing: true,
-        },
-      ])(
-        'updates the templateStatus to NOT_YET_SUBMITTED when global proofing is $globalProofing and client proofing is $clientProofing',
-        async ({ clientProofing, globalProofing }) => {
-          const { templateRepository, mocks } = setup(globalProofing);
-
-          await templateRepository.setLetterValidationResult(
-            { owner: 'template-owner', id: 'template-id' },
-            'file-version-id',
-            true,
-            ['personalisation', 'parameters'],
-            ['csv', 'headers'],
-            clientProofing
-          );
-
-          expect(mocks.ddbDocClient).toHaveReceivedCommandWith(UpdateCommand, {
-            TableName: 'templates',
-            Key: { id: 'template-id', owner: 'template-owner' },
-            UpdateExpression:
-              'SET #templateStatus = :templateStatus , #updatedAt = :updatedAt , #personalisationParameters = :personalisationParameters , #testDataCsvHeaders = :testDataCsvHeaders',
-            ConditionExpression:
-              '#files.#file.#version = :version and not #templateStatus in (:templateStatusDeleted, :templateStatusSubmitted)',
-            ExpressionAttributeNames: {
-              '#testDataCsvHeaders': 'testDataCsvHeaders',
-              '#file': 'pdfTemplate',
-              '#files': 'files',
-              '#personalisationParameters': 'personalisationParameters',
-              '#templateStatus': 'templateStatus',
-              '#updatedAt': 'updatedAt',
-              '#version': 'currentVersion',
-            },
-            ExpressionAttributeValues: {
-              ':testDataCsvHeaders': ['csv', 'headers'],
-              ':personalisationParameters': ['personalisation', 'parameters'],
-              ':templateStatus': 'NOT_YET_SUBMITTED',
-              ':templateStatusDeleted': 'DELETED',
-              ':templateStatusSubmitted': 'SUBMITTED',
-              ':updatedAt': '2024-12-27T00:00:00.000Z',
-              ':version': 'file-version-id',
-            },
-          });
-        }
-      );
+    describe('when proofing is disabled for the client', () => {
+      test('updates the templateStatus to NOT_YET_SUBMITTED when client proofing is turned off', async () => {
```

Comment:
```suggestion
      test('updates the templateStatus to NOT_YET_SUBMITTED', async () => {
```
dont reset your approvals for this but if you end up making another commit, this is already in the desc so dont think you need it here too